src/main/kotlin/io/github/moulberry/notenoughupdates/miscgui/DynamicLightItemsEditor.kt

```diff
@@ -0,0 +1,232 @@
+            drawTexturedModalRect(guiLeft + 27 + i % 9 * 18, guiTop + 24 + j * 18, 7, 17, 18, 18)
+
+            val itemStack = resolveItemStack(item) ?: return
+            Utils.drawItemStack(itemStack, guiLeft + 28 + i % 9 * 18, guiTop + 25 + j * 18)
+
+            if (mouseX >= guiLeft + 27 + i % 9 * 18 && mouseX <= guiLeft + 45 + i % 9 * 18) {
+                if (mouseY >= guiTop + 24 + j * 18 && mouseY <= guiTop + 42 + j * 18) {
+                    hoveredItem = item
+                    val tooltip = itemStack.getTooltip(Minecraft.getMinecraft().thePlayer, false)
+                    Utils.drawHoveringText(tooltip, mouseX, mouseY, width, height, -1)
+                }
+            }
+
+            if (itemSelected != null && itemSelected.equals(item)) {
+                // Save the position, so when we render the selected box its renders on top of everything
+                selectedPosition = Pair(guiLeft + 24 + i % 9 * 18, guiTop + 21 + j * 18)
+            }
+
+            i++
+            j += if (i % 9 == 0) 1 else 0
+        }
+
+        stackToRender = hoveredItem
+
+        GlStateManager.color(1f, 1f, 1f, 1f)
+        Minecraft.getMinecraft().textureManager.bindTexture(widgets)
+        drawTexturedModalRect(selectedPosition.first, selectedPosition.second, 0, 22, 24, 24)
+
+        fontRendererObj.drawString("Add Held Item", guiLeft + 27, guiTop + ySize - 26, 4210752)
+        fontRendererObj.drawString("Remove Item", guiLeft + 130, guiTop + ySize - 26, 4210752)
+
+        GlStateManager.color(1f, 1f, 1f, 1f)
+        Minecraft.getMinecraft().textureManager.bindTexture(help)
+        Utils.drawTexturedRect((guiLeft + xSize + 3).toFloat(), guiTop.toFloat(), 16F, 16F, GL11.GL_NEAREST)
+        if (mouseX >= guiLeft + xSize + 3 &&
+            mouseX <= guiLeft + xSize + 19 &&
+            mouseY >= guiTop &&
+            mouseY <= guiTop + 16) {
+            var tooltip: MutableList<String> = mutableListOf()
+            tooltip.add("§bDynamic Light Item Editor")
+            tooltip.add("§eWhat is this?")
+            tooltip.add("§eNEU makes use of OptiFine's feature of certain items")
+            tooltip.add("§eemitting dynamic light. By default OptiFine only implements")
+            tooltip.add("§ethis feature for a select few minecraft items.")
+            tooltip.add("")
+            tooltip.add("§eThis editor however, allows you to add specific skyblock")
+            tooltip.add("§eitems that will emit dynamic light when held. Simply hold the")
+            tooltip.add("§eitem you wish to add, then open this menu again and click")
+            tooltip.add("§e'Add Held Item', now if you have OptiFine installed and the")
+            tooltip.add("§edynamic lights option enabled, the added items will emit light!")
+            tooltip.add("")
+            tooltip.add("§eTo remove an item, click the item in this menu and click")
+            tooltip.add("§ethe 'Remove Item' button in the bottom right.")
```

Comment:
You can replace all that with 
```suggestion
            val tooltip = listOf(
                "§bDynamic Light Item Editor",
                "§eWhat is this?",
                "§eNEU makes use of OptiFine's feature of certain items",
                "§eemitting dynamic light. By default OptiFine only implements",
                "§ethis feature for a select few minecraft items.",
                "",
                "§eThis editor however, allows you to add specific skyblock",
                "§eitems that will emit dynamic light when held. Simply hold the",
                "§eitem you wish to add, then open this menu again and click",
                "§e'Add Held Item', now if you have OptiFine installed and the",
                "§edynamic lights option enabled, the added items will emit light!",
                "",
                "§eTo remove an item, click the item in this menu and click",
                "§ethe 'Remove Item' button in the bottom right.",
            )
```